src/models/user/user.schema.ts

```diff
@@ -21,5 +25,7 @@ const userSchema = new Schema<IUser>(
   } as SchemaOptionsWithPojoToMixed
 );
 
+userSchema.statics.findOneByEmail = findOneByEmail;
+userSchema.statics.createUser = createUser;
```

Comment:
Notice here that I need to add my model-level controller functions to this `statics` property.